plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -73,8 +75,8 @@ public void execute(IProgressMonitor monitor) throws InvocationTargetException,
       installWebFacet(facetedProject, progress.newChild(10));
       
       // must happen after other two facets because the appengine facet requires them
-      AppEngineFacet.installAppEngineFacet(facetedProject, progress.newChild(10));
-      AppEngineFacet.installAppEngineRuntime(facetedProject, progress.newChild(20));
+      FacetInstallDelegate.installAppEngineFacet(facetedProject, progress.newChild(10));
```

Comment:
In `CreateMavenBasedAppEngineStandardProject` a different number of ticks is passed in. It looks inconsistent.